src/app/common/api.ts

```diff
@@ -0,0 +1,53 @@
+import axios from "axios";
+import { baseAPIURL } from "../../shared/constants";
```

Comment:
Can we move the base API URL to globals and call it like __BASE_API_URL__?

Reply:
Yes I can do that